components/collective-page/index.js

```diff
@@ -183,8 +183,6 @@ class CollectivePage extends Component {
   renderSection(section) {
     switch (section) {
       // v2 transition - these standalone sections will be removed from render if feature flag is active
```

Comment:
Shouldn't we remove that comment? Doesn't seems to be true anymore. Based on what we defined, Conversations and Updates should be two distinct sections under the "Connect" category.